test/spec/testUtils.spec.ts

```diff
@@ -0,0 +1,169 @@
+var utils;
+
+// This needs to be cleaned, and maybe externalized into a small testing
+// framework utils. But then we'd need to find a name for it.
+utils = angular.module('testUtils', []);
```

Comment:
alright, good you added it - there is some very useful stuff included. however, a lot of things (dealing with mouse, classes, dom elements, ..) should be now supported with protractor